### PR TITLE
Troubleshooting page

### PR DESCRIPTION
**Pull request by mike-ferguson**
This page serves as a compilation of common errors that users may encounter in the submission process, and their solutions

### CODE REVIEW

**Comment by mike-ferguson** [comment]
example question: 
Screenshot 2024-03-13 at 10 00 14 AM

File: benchmarks/templates/benchmarks/tutorials/tutorial_troubleshooting.html [hunk 1]
```diff
@@ -0,0 +1,151 @@
...
+                solutions for a variety of problems, with the hope that users can quickly find the help they need.
+            </p>
+            <p class="benefits_info is-size-5-mobile">
+                We also continually update this guide based on user feedback and emerging issues, ensuring it remains
+                a valuable tool for resolving challenges efficiently.
+            </p>
+
+            <p class="benefits_info is-size-5-mobile">
+                If your issue is not listed here, please feel free to open a Github issue, check out the community page,
+                or contact the Brain-Score team directly via slack or email; we are always happy to help!
```
Comment (mschrimpf) [inline benchmarks/templates/benchmarks/tutorials/tutorial_troubleshooting.html:32]: link the community tab here which has links for all of these

File: benchmarks/templates/benchmarks/tutorials/tutorial_troubleshooting.html [hunk 2]
```diff
@@ -0,0 +1,151 @@
...
+            </p>
+        </div>
+    </div>
+</div>
+
+<!-- Each Div is a section itself:-->
+<div class="faq_box leaderboard-table-component">
+    <button class="toggle-button is-size-3-mobile" aria-expanded="false">
+      <span class="benefits_heading"><h3 class="benefits_heading is-size-3-mobile">Model Submissions </h3></span>
```
Comment (mschrimpf) [inline benchmarks/templates/benchmarks/tutorials/tutorial_troubleshooting.html:41]:
```suggestion
      <span class="benefits_heading"><h3 class="benefits_heading is-size-3-mobile">Model Submissions</h3></span>
```

File: benchmarks/templates/benchmarks/tutorials/tutorial_troubleshooting.html [hunk 3]
```diff
@@ -0,0 +1,151 @@
...
+      <span class="icon is-small"><i class="fas fa-caret-down icon-gradient"></i></span>
+    </button>
+    <div class="content is-hidden">
+        <p class="benefits_info is-size-5-mobile question_text has-text-weight-bold no_top_header">
+            1) My submission didn't work. What happened?
+        </p>
+        <p class="benefits_info is-size-5-mobile shorter no_top">
+            <span class="is-italic">Description:</span> This is the most common question we get, and our
+            <a href="http{% if request.is_secure %}s{% endif %}://{{ request.get_host }}/faq">FAQ</a> gives a nice
+            overview of what could have happened.
```
Comment (mschrimpf) [inline benchmarks/templates/benchmarks/tutorials/tutorial_troubleshooting.html:51]:
```suggestion
            <a href="http{% if request.is_secure %}s{% endif %}://{{ request.get_host }}/faq">FAQ</a> provides an
            overview of what could have happened.
```

File: benchmarks/templates/benchmarks/tutorials/tutorial_troubleshooting.html [hunk 4]
```diff
@@ -0,0 +1,151 @@
...
+            <span class="is-italic">Cause:</span> You are trying to save your weights to a local (relative) path, or
+            for some reason the server cannot locate your <span class="special_format">.pth</span> (weights) file. Sometimes, your working path
+            on your machine might not be the same as the working path our server uses to execute runs.
+        </p>
+        <p class="benefits_info is-size-5-mobile shorter no_top">
+            <span class="is-italic">Fix:</span> We have written code that allows users to download from S3 buckets
+             <a href="https://github.com/brain-score/vision/blob/master/brainscore_vision/model_helpers/s3.py">here</a>,
+             specifically  the <span class="special_format">load_file_from_s3</span> function.
+             Please use this code in place of your own if you are hosting your weights on S3. We hope to add support
+             for other cloud storage, like Google Drive, very soon.
```
Comment (mschrimpf) [inline benchmarks/templates/benchmarks/tutorials/tutorial_troubleshooting.html:104]:
```suggestion
             Please use this code in place of your own if you are hosting your weights on S3. We hope to eventually add support
             for other cloud storage, like Google Drive.
```

**Review by mschrimpf — APPROVED** [review]

**Pull request merged**